src/components/toplayer/TopLayer.jsx

```diff
@@ -38,7 +40,7 @@ const TopLayer = props => {
 
   return (
     <div {...additionalProps} className={topLayerClassName}>
-      <div className="sg-toplayer__close">
+      <div className="sg-toplayer__close" onClick={onClose}>
```

Comment:
maybe hide 'x' when no defined onClose function? If no pls create default props for onClose, coz i current state will try call undefined()

Reply:
@matikrk Please have a look now.